Documentation/spelling_wordlist.txt

```diff
@@ -187,6 +187,7 @@ artii
 asm
 assignees
 au
+auth
```

Comment:
Can you please fold this change with the commit that introduces the word in the documentation?

Reply:
of course, done